### PR TITLE
DTGB-162: Added styling for the breadcrumbs component.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss [hunk 1]
```diff
@@ -2,46 +2,42 @@
   ul,
   ol {
     display: none;
-    margin: .5rem auto .5rem -.35rem;
+    line-height: normal;
```
Comment (delrueba) [inline styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss]: Is browser dependent.

**Review by Jeroen005 — APPROVED** [review]

**Review by delrueba — CHANGES_REQUESTED** [review]
The wireframes for the 'collapsing' breadcrumbs on mobile aren't finished yet and the old implementation is not accessible.

If a styleguide script is responsible for this, can you exclude it and add a todo?

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss]: Turns out this wasn't necessary. I removed it

**Comment by meirege** [comment]
@delrueba I removed the old breadcrumbs JS. If necessary later we can revert my commit.

**Review by delrueba — APPROVED** [review]

**Pull request merged**